python/setup.py

```diff
@@ -159,11 +168,20 @@
 ]
 
 for e in extensions:
-    e.exclude = exc_list
+    # TODO: this exclude is not working, need to research way to properly
+    # exclude files for parallel build. See issue
+    # https://github.com/rapidsai/cuml/issues/2037
+    # e.exclude = exc_list
     e.cython_directives = dict(
```

Comment:
Just so I understand... does this mean that with the singlegpu flag, everything works normally just not parallelized over the extensions?

Reply:
Correct, this is a little unfortunate, opened github issue #2037 to track this. Not a huge deal since the bulk of users and CI build with libcumlprims, but a nuisance for our users of singlegpu. That said, it adds like a couple of minutes of build at worst (less maybe), so not a huge priority IMO.